docusaurus.config.js

```diff
@@ -325,6 +355,8 @@ const config = {
         'yaml',
         'csharp',
         'dart',
+        // Leave php disabled until this issue is fixed upstream: https://github.com/PrismJS/prism/issues/2769
+        //'php'
```

Comment:
I noticed that the Python examples don't seem to be getting syntax highlighting either. They do on docs.devcycle.com though, even though it doesn't seem to have been in this list to begin with. Any idea what's happening there?

Reply:
hmm - i'll add it to the list and see what happens. Python only recently got support

https://github.com/FormidableLabs/prism-react-renderer/pull/226